pkg/cmd/server/api/v1/conversions.go

```diff
@@ -88,7 +88,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 				obj.NetworkConfig.MTU = 1450
 			}
 			if len(obj.IPTablesSyncPeriod) == 0 {
-				obj.IPTablesSyncPeriod = "5s"
```

Comment:
No. On the node, k8s periodically syncs proxy iptables rules (to ensure it's not overwritten by other tools or by hand) and the frequency of this sync loop is driven by this IPTablesSyncPeriod config. Originally, k8s started with 5s and now they changed it to 30s.